gcp/rakefiles/secrets.rb

```diff
@@ -92,17 +92,17 @@ def self.set_secrets(collected_secrets)
       else
         populated_secrets = {}
         secrets.each do |secret_name|
-          if ENV[secret_name.upcase].to_s.empty?
+          if ENV["TF_VAR_#{secret_name}"].to_s.empty?
             if secret_name.start_with?("key_")
-              key = OpenSSL::Cipher.new("aes-256-cfb").encrypt.random_key
-              secret_value = Base64.encode64(key).chomp
+              key = OpenSSL::Cipher::AES256.new.encrypt.random_key
+              secret_value = Base64.strict_encode64(key)
```

Comment:
Are there like 2 different RFCs how to do Base64? RFC 2045 & RFC 4648? huh... :)

Reply:
Yep. With previous implementation, there was a high chance that invalid Base64 data is produced when special characters is present in key's binary escaped string.